recipes/pyprism/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pyPRISM" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "bc343b8827bdcfbe725639674ceca3e3c7a51bf352b7987a9ae4af85148c6deb" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy >=1.8.0
+  run:
+    - python
+    - numpy
+    - scipy
+    - pint
+
+test: 
+  requires:
+    - pytest
+    - numpy 
```

Comment:
This should be installed at runtime (so you don't need to include the run requirements).